services/events.service.js

```diff
@@ -30,61 +29,53 @@ class EventsService extends BaseService {
       /* eslint-disable no-console */
       // console.error('Error creating an event: ', err)
       /* eslint-enable */
-      response = this.getErrorResponse(errorMessage);
+      response = this.getErrorResponse(errorMessage)
     }
 
-    return response;
+    return response
   }
 
   async doList(eventParams) {
     let allEvents = []
     let response
 
-    const {
-      year,
-      withAttendees,
-      headquarterId,
-      showAllStatus
-    } = eventParams
-
-    if (!year) {
-      // TODO: Create a constants object and replace this message
-      throw new Error('Missing parameter')
-    }
+    const { year, withAttendees, headquarterId, showAllStatus } = eventParams
 
     try {
+      let rootQuery = this.collection
 
-      let rootQuery = this.collection.where('year', '==', parseInt(year, 10))
+      if (year) {
+        rootQuery.where('year', '==', parseInt(year, 10))
```

Comment:
en mi ignorancia de mongo, no deberia ser asi ?
```suggestion
        rootQuery.where('year', '===', parseInt(year, 10))
```

Reply:
Para este caso, se leyó de cual manera era mejor y optamos por sólo dos =
De igual manera quedamos atentas a recomendaciones